src/main/java/org/galatea/starter/domain/IexHistoricalPrice.java

```diff
@@ -0,0 +1,20 @@
+package org.galatea.starter.domain;
+
+import java.math.BigDecimal;
+import java.util.Date;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class IexHistoricalPrice {
+
+  private BigDecimal close;
+  private BigDecimal high;
+  private BigDecimal low;
+  private BigDecimal open;
+  private String symbol;
+  private long volume;
+  private String date;
```

Comment:
It seems that the IEX API historical prices and symbols endpoints return dates as strings: 

![image](https://user-images.githubusercontent.com/92825615/157712062-ddb0f567-7c62-4e35-bef9-a1ad23900cc5.png)

When I was originally hitting the endpoint, I was getting a lengthy date number (e.g. "updated" key). Is this milliseconds from Jan 1970? 

Let me know if you'd like me to change date back to the Date type or parse the date object to a string instead of receiving the data from the API as a string.